docs/testing.rst

```diff
@@ -4,8 +4,119 @@ Testing
 Overview
 --------
 
+The ``django-socio-grpc`` is a powerful library that serves as a layer on top of the Django Rest Framework (DRF). It is designed to enhance the capabilities of Django and DRF, providing additional functionality and features.
+
+When it comes to testing, ``django-socio-grpc`` follows the same conventions as Django and DRF. It utilizes the ``unittest`` library, a built-in Python module for testing. This allows for the creation of robust and comprehensive unit tests, ensuring the reliability and stability of the codebase.
+
+The following sections will guide you through the process of writing and running tests for ``django-socio-grpc``.
+
 Writing Tests
 -------------
 
+Let's imagine a dummy service for a model Post
+
+  .. code-block:: python
+    
+    # quickstrart/services.py
+    from django_filters.rest_framework import DjangoFilterBackend
+    from django_socio_grpc.generics import AsyncModelService
+
+    from quickstart.models import Post
+    from quickstart.serializer import PostProtoSerializer
+
+    # AsyncModelService implements basic crud functionnalities
+    class PostService(AsyncModelService):
+        queryset = Post.objects.all()
+        serializer_class = PostProtoSerializer
+        filter_backends = [DjangoFilterBackend]
+        filterset_fields = ["headline"]
+
+
+The following is a basic structure for your testing class:
+
+  .. code-block:: python
+
+    # quickstrart/tests.py
+    from django.test import TestCase
+    from .post.services.post_service import PostService
+    from .post.grpc import post_pb2
+    from .post.grpc.post_pb2_grpc import (
+        PostControllerStub,
+        add_PostControllerServicer_to_server,
+    )
+    from .grpc_test_utils.fake_grpc import FakeFullAIOGRPC
+    
+    class TestPost(TestCase):
+        def setup(self):
+            self.fake_grpc = FakeFullAIOGRPC(
+                add_PostControllerServicer_to_server,
+                PostService.as_servicer(),
+            )
+
+        def tearDown(self):
+            self.fake_grpc.close()
+
+Now, let's dissect the ``TestPost`` class:
+
+- We recommend using ``TestCase`` from ``django.test`` for testing your service.
+- The ``setup`` method should instantiate a ``FakeFullAIOGRPC``.
+- ``FakeFullAIOGRPC`` is initialized with two parameters: a function to add a servicer to a server, and the servicer itself. In this case, ``add_PostControllerServicer_to_server`` is the function generated by gRPC that adds a ``PostService`` servicer to a server.
```

Comment:
This is too compact and not enought explicit
make a sub list a two element to explain add_PostControllerServicer_to_server and what is a servicer.
Add a link to the grpc doc: https://grpc.io/docs/languages/python/basics/ saying to look for Starting the server part of the document